src/spade/container/dom.cljs

```diff
@@ -0,0 +1,61 @@
+(ns spade.container.dom
+  "The DomStyleContainer renders styles into DOM elements. References to those
+   elements are stored in a `styles` atom, or `*injected-styles*` if that is
+   not provided. Similarly, if no `target-dom` is provided, the `document.head`
+   element is used."
+  (:require [spade.container :refer [IStyleContainer]]))
+
+(defonce ^:dynamic *injected-styles* (atom nil))
```

Comment:
Is this `*injected-styles*` shared between containers ? If so would that work for the case where you have the same style being injected into two different target containers ?

Reply:
`*injected-styles*` is not shared! We may even be able to get rid of it, but basically it is the *default* storage atom for the *default* DOM container. In other words, every container created by `(dom/create-container)` (IE without any args) will share this atom and store style elements in `<head>`. Any container created by `(dom/create-container element)` will have its own styles storage atom created by default. Finally, you can provide your own storage atom if you wish for some reason.